src/NUnitFramework/tests/Constraints/ThrowsConstraintTests.cs

```diff
@@ -53,7 +53,7 @@ public void SetUp()
     }
 
     [TestFixture]
-    public class ThrowsConstraintTest_InstanceOfType : ConstraintTestBase
+    public class ThrowsConstraintTest_InstanceOfType : ThrowsConstraintTestBase
```

Comment:
@ZenonBuratta Can you help me understand this change?

Reply:
I also wondered about this, so I took a look at the code. All other classes in this file inherits from `ThrowsConstraintTestBase`, and the only difference between `ConstraintTestBase` and `ThrowsConstraintTestBase` is the following:
https://github.com/nunit/nunit/blob/7b87bdc7b48ee1b13023dd282efea449c2d66b81/src/NUnitFramework/tests/Constraints/ConstraintTestBase.cs#L72-L74
vs
https://github.com/nunit/nunit/blob/f94ea919fe344fc656c84052d885cbd3c4aedc71/src/NUnitFramework/tests/Constraints/ThrowsConstraintTests.cs#L131-L133
Hence the assert is changed from `Is.EqualTo` to `Does.StartWith`. Perhaps a bit overkill to have two so similar classes, but that is another issue 😄.

Reply:
Yes @mikkelbu. The change of base class is the ThrowsConstraintTests uses a Does.StartWith. This is because of the new stack trace that comes with the exception and due to not wanting to make the tests become fragile, I only want to check the message.